### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by ezraermy**
This pull request creates a database using PostgreSQL
The implementation includes the following changes:
- Query multiple tables
- Use primary key & foreign key mechanisms for joining tables.
- Prepare queries that answer analytical questions.
- Screenshots:
                         1. 1st_half.png
                         2. 2nd_half.png 
                         3. digimon_pokemon_queries.png
                         4. species_unspecified_queries.png
                         5. transaction_create_savepoint.png
                         6. transaction_delete_rollback.png
                         7. transaction_update_species.png
                        8. write_queries1.png
                        9. write_queries2.png
                        10. queries_multiple_table1.png 
                        11. queries_multiple_table2.png 

## Other essential requirements:
- Use Gitflow.
- Documented in a professional way.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,4 +7,21 @@ CREATE TABLE animals(
...
+);
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name TEXT
+);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN species_id INTEGER;
```
Comment (BregornOriginal) [inline schema.sql:25]: Good work adding columns to the animal's table 💯 

- As it's on the requirements for this project, please add the corresponding foreign key referencing species table `species(id)`

File: schema.sql [hunk 2]
```diff
@@ -7,4 +7,21 @@ CREATE TABLE animals(
...
+
+ALTER TABLE animals ADD COLUMN owner_id INTEGER;
```
Comment (BregornOriginal) [inline schema.sql]: Good work adding columns to the animal's table 💯 

- As it's on the requirements for this project, please add the corresponding foreign key referencing owners table `owners(id)`

File: queries.sql [hunk 3]
```diff
@@ -46,3 +46,41 @@ SELECT species, AVG(escape_attempts) AS avg_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
```
Comment (BregornOriginal) [inline queries.sql:49]: Good work creating the queries 💯 

- Kindly as you added screenshots to your project, add them also in the Pull Request description

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @ezraermy  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

cat-driving-serious

## Highlights

✔️ README looks professional 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (ezraermy) [reply queries.sql:49]: @BregornOriginal Thanks a lot for your time and for providing helpful feedback.

File: schema.sql [hunk 4]
```diff
@@ -24,4 +24,15 @@ ALTER TABLE animals DROP COLUMN species;
```
Comment (HAMMAS-SALEEM) [inline schema.sql:24]: image

- [ ] As suggested by the previous reviewer, kindly add all the screenshots in the pull request summary to follow the project requirements 👍

Comment (HAMMAS-SALEEM) [inline schema.sql:24]: **[OPTIONAL]**

- [ ] Kindly make your readme more descriptive by removing unnecessary details such as section descriptions, NOTES and details not related to this project 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @ezraermy :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 4 above)
Comment (ezraermy) [reply schema.sql:24]: @HAMMAS-SALEEM Thank you for your feedback. However, It is not clear which part of my readme file is not required. I already removed all unnecessary sections that are not essential at this stage. I also don't understand why a short description of the project is not required. This has been accepted by almost all reviewers.

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**